simics-rs/simics-macro/src/interface/mod.rs

```diff
@@ -754,7 +754,7 @@ impl CInterface {
                     .join("py-typemaps.c"),
             )
             .arg(interface_subdir.join(&header_name))
-            .print_args()
+            // .print_args()
```

Comment:
Don't leave in commented out code. Either make it conditional on some debug flag/verbosity, or remove it.